src/MicrosoftAuthFlow.ts

```diff
@@ -25,18 +25,37 @@ async function retry (methodFn, beforeRetry, times) {
   }
 }
 
+type MicrosoftAuthFlowOptions = {
+  flow: 'live' | 'sisu' | 'msal',
+  authTitle?: string | undefined,
+  relyingParty?: string | undefined,
+  password?: string | undefined,
+  fetchEntitlements?: boolean | undefined,
+  fetchProfile?: boolean | undefined,
+  fetchCertificates?: boolean | undefined
+}
+
 class MicrosoftAuthFlow {
-  constructor (username = '', cache = __dirname, options, codeCallback) {
+  username: string
+  options: MicrosoftAuthFlowOptions
+  xbl?: typeof XboxTokenManager | undefined
```

Comment:
xbl through to doTitleAuth shouldn't be marked as public properties, they're internal and are not part of the API so shouldn't be documented in types.